Source/GUI/Qt/checkerwindow.cpp

```diff
@@ -148,9 +148,15 @@ void CheckerWindow::set_web_view_content(QString& html)
     QObject::connect(main_view, SIGNAL(loadProgress(int)), progress_bar->get_progress_bar(), SLOT(setValue(int)));
     QObject::connect(main_view, SIGNAL(loadFinished(bool)), this, SLOT(create_web_view_finished(bool)));
 
+#if QT_VERSION >= 0x050700
+    QUrl url = QUrl("");

```

Comment:
Wow... OK.
And is it also broken if you set "" in Qt 5.6-?
and isn't [QUrl()](https://doc.qt.io/qt-5/qurl.html#QUrl) better?